src/current/v24.3/configure-logs.md

```diff
@@ -637,6 +637,8 @@ With this logging configuration:
 - If at any point the accumulated message size of buffer flushes (as triggered by reaching either the configured `max-staleness` or `flush-trigger-size` value) exceeds `100MiB` (the `max-buffer-size` setting), all new incoming log messages received are dropped until the accumulated message size in the buffer once more falls below this value.
 - For the `HEALTH` [log channel]({% link {{ page.version.version }}/logging-overview.md %}#logging-channels) only, override the `file-defaults` value of `20s` for `max-staleness`, instead flushing messages to the log file within up to `2s`.
 
+{% include_cached new-in.html version="v24.3" %}If CockroachDB is managed using `systemd` and log buffering is enabled, logs are flushed automatically when CockroachDB starts.
```

Comment:
This doesn't reflect the intention of the PR in https://github.com/cockroachdb/cockroach/pull/125971/files. Previously, without the change, logs are flushed regardless whenever CockroachDB starts. Now, with that change, logs are only flushed when CockroachDB is managed using systemd.